app/src/main/res/values-nl/strings.xml

```diff
@@ -1,152 +1,142 @@
-<!--
```

Comment:
Don't Remove these

Reply:
Sorry and i resolved the conflict so what is wrong ?!

Reply:
solved it.. hopefully